packages/flutter/lib/src/material/progress_indicator.dart

```diff
@@ -303,6 +333,14 @@ class LinearProgressIndicator extends ProgressIndicator {
   /// {@endtemplate}
   final double? minHeight;
 
+  /// The border radius of the indicator.
+  /// By default it is [BorderRadius.zero], which produces a rectangular indicator.
+  final BorderRadius indicatorBorderRadius;
+
+  /// The border radius of the background indicator.
+  /// By default it is null, which produces a rectangular track.
```

Comment:
```suggestion
  /// 
  /// By default it is null, which produces a rectangular track.
```
This is a non-nullable parameter. How does it default to null? :)

Reply:
ooops :P

Reply:
Actually, I was playing with a few things and forgot to update the documentation. I was checking if I could "auto add a borderRadius" but my idea was bad and it is better to allow the user to break things. IndicatorBorderRadius = 100 + background = 0 result in a weird scenario I was trying to avoid, but I'm letting the user decide.